pyanaconda/dbus_addons/Makefile.am

```diff
@@ -0,0 +1,24 @@
+# dbus_addons/Makefile.am for anaconda
```

Comment:
You also need to add these Makefiles to the confiure.ac file.

Reply:
You need to change `pyanaconda/Makefile.am` to add `SUBDIRS` there.

Reply:
Good catch - fixed!
But other than that - what insanity is this ? I kinda understand the idea that you have a Makefile.am in each folder, that specifies which files in the folder (if any) should be taken into account during build & defines any build-related subfolders (if any).

This should be enough for autotools to crawl the source tree, going over individual Makefile.am files. But it needs a static list of **all** the Makefiles as well ?? That's just totally redundant, fragile   and insane. :P

Reply:
For that I could say only: "Die autotools..." <-- choose your language.

Reply:
It's "Die **A**utotools" in German. :)